agent/runner/actions/mysql_explain_action_test.go

```diff
@@ -167,13 +173,9 @@ func TestMySQLExplain(t *testing.T) {
 			Dsn:          "pmm-agent:pmm-agent-wrong-password@tcp(127.0.0.1:3306)/world",
 			OutputFormat: agentpb.MysqlExplainOutputFormat_MYSQL_EXPLAIN_OUTPUT_FORMAT_DEFAULT,
 		}
-		a := NewMySQLExplainAction("", time.Second, params)
-		ctx, cancel := context.WithTimeout(context.Background(), a.Timeout())
-		defer cancel()
-
-		_, err := a.Run(ctx)
-		require.Error(t, err)
+		a, err := NewMySQLExplainAction("", time.Second, params)
 		assert.Regexp(t, `Query to EXPLAIN is empty`, err.Error())
```

Comment:
assert has `ErrorContains` to check if error message contains substring. in case of no error this test might panic. but in this case it seems like `assert.EqualError` would be enough

Reply:
Used `ErrorContains`